SingularityUI/app/models/Request.coffee

```diff
@@ -121,15 +121,21 @@ class Request extends Model
                 @unpause().done callback
 
     promptRun: (callback) =>
+                  
         vex.dialog.prompt
-            message: runTemplate id: @get "id"
+            input: runTemplate id: @get "id"
```

Comment:
Yea, (as far as I can tell) this allows me to specify additional inputs (and the id/order) - since the message object before only loaded the additional command line input